xbmc/filesystem/File.cpp

```diff
@@ -360,7 +360,12 @@ bool CFile::Open(const CURL& file, const unsigned int flags)
       return false;
     }
 
-    if (m_pFile->GetChunkSize() && !(m_flags & READ_CHUNKED))
+    constexpr int64_t len = 200 * 1024 * 1024; // 200 MB
```

Comment:
How did you decide to use 200MB?

Reply:
It's only assumed that the videos that people usually watch (movies/shows) normally weigh more than 200 MB. In any case, if this is not fulfilled, nothing bad happens: only `CFileStreamBuffer` is not used (same as before) and it's read from USB/disk with 4K chunks.